### PR TITLE
Semantics, seo and accessibility

### PR DESCRIPTION
**Pull request by Hornebom**
First of all thank you for putting so much effort in this project. As a user I want to support you guys in making it even better. I decided to improve the lighthouse scores for seo and accessibility and align the markup to semantic standards. 

Please understand the pull request as a starting point. If you agree, I'll refactor more parts of the application.

Semantics:
- refactor header to use `header` tag
- refactor `Page` to use `main` tag and have a `h1` with a title
- refactor `Feed` to use `ul` and `li`

Seo:
- add a `h1` title to `Page`
- adds open graph meta tags to `index.html` (also adds a share image)
- using `Helmet` to set the page title

Accessibility:
- creates a `VisuallyHidden` component to add descriptive text to titles and interactive elements 

A note on `App.js`
As I wanted to set a `title` for each page I refactored how routes are defined in `App.js`. I decided to use a `routes.js` and simply map over each route. In that way the App component itself isn't bloated with repeating code. I was not using the routes defined in `constants.js` as they were not used anywhere and seem to be incomplete.

I'm looking forward to your feedback and willing to update this pr to your needs.

### CODE REVIEW

**Comment by andrevenancio** [comment]
There's a couple of conflicts to fix here. Also would be easier if you could make smaller PR's so its easier to review! But fix the conflicts and I'll have a look.

File: public/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>https://www.hicetnunc.xyz</loc>
+    </url>
+    <url>
+        <loc>https://leerob.io/hdao</loc>
```
Comment (andrevenancio) [inline public/sitemap.xml]: LeeRob.io?

Comment (Hornebom) [reply public/sitemap.xml]: hahahha, good catch

**Comment by Hornebom** [comment]
@andrevenancio conflicts resolved. I could also split up the pr into three smaller ones, if that helps

**Comment by andrevenancio** [comment]
Smaller ones would help! but you can just fix the conflicts and next time do it in smaller PR's

**Comment by Hornebom** [comment]
@andrevenancio resolved again 👍

**Review by andrevenancio — APPROVED** [review]

**Pull request merged**